backend/dataall/modules/notifications/tasks/weekly_digest_reminder.py

```diff
@@ -0,0 +1,261 @@
+            # function _map_groups_to_resource_bundles maps each team name : resource bundle
+            # Equivalent to calling
+            # _map_groups_to_resource_bundles(list_of_notifications=pending_share_notification_resources,
+            #                                 resource_bundle_type="share_object_notifications")
+            # _map_groups_to_resource_bundles(list_of_notifications=unhealthy_share_objects_notification_resources,
+            #                                 resource_bundle_type="share_object_notifications") ....
+
+            for notification_resources, resource_bundle_type in resources_type_tuple:
+                _map_groups_to_resource_bundles(list_of_notifications=notification_resources, resource_bundle_type=resource_bundle_type)
+
+            for group, resource_bundle in group_name_to_resource_bundle_map.items():
+                email_body = _construct_email_body(resource_bundle)
+                log.debug(email_body)
+                subject = 'Attention Required | Data.all weekly digest'
+                try:
+                    SESEmailNotificationService.create_and_send_email_notifications(subject=subject, msg=email_body,
+                                                                                    recipient_groups_list=[group])
+                except Exception as e:
+                    log.error(f"Error occurred in sending email while weekly reminder task due to: {e}")
+                    task_exceptions.append(f"Error occurred in sending email while weekly reminder task due to: {e}")
+    except Exception as e:
+        log.error(f"Error occurred while running the weekly reminder task: {e}")
+        task_exceptions.append(f"Error occurred while running the weekly reminder task: {e}")
+    finally:
+        if len(task_exceptions) > 0:
+            log.info("Sending email notifications to the admin team")
+            AdminNotificationService().notify_admins_with_error_log(
+                process_error="Error occurred while running the weekly reminder task",
+                error_logs=task_exceptions,
+                process_name="Weekly reminder task"
+            )
+
+
+def _construct_email_body(resource_bundle: NotificationResourceBundle):
+    msg_heading = """
+    Dear Team, <br><br>
+    
+    This email contains data.al resources where you need to take some actions. For resources which are in unhealthy state we request you to take actions ASAP so as to minimize any disruptions.<br><br>
+    
+    <b>Helpful Tips:</b><br><br>
+    For shares which are in unhealthy state, you can re-apply share by clicking on the "Reapply share" button <br>
+    For environments and datasets which are in unhealthy state, you can go to the AWS account and check the stack associated with that environment/dataset and check the root cause of the stack. Once you address the root cause issue, you can click on "Update Stack" on the Stack Page. <br><br><br> 
```

Comment:
nit: specify `"Update Stack" on the Stack Page of the data.all resource in the data.all UI`